controllers/services/auth/resources/admingroup-role.tmpl.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: admingroup-role
+  namespace: {{.DSCI.Spec.ApplicationsNamespace}}
```

Comment:
Does this Role not need to be a ClusterRole to give access to a Cluster resource?

Reply:
Yes :facepalm: You're right, let me fix that.

Reply:
Thanks, I pushed the change. I left the namespaces role as a 'bootstrap' kinda file because we know the content will be changing as a result of [The permissions issue](https://issues.redhat.com/browse/RHOAIENG-14235).

```
$ oc adm policy who-can --all-namespaces  patch auth 
resourceaccessreviewresponse.authorization.openshift.io/<unknown> 

Namespace: <all>
Verb:      patch
Resource:  auths.services.platform.opendatahub.io
<SNIP>
Groups: aTestAdminGroup
        odh-admins
        system:cluster-admins
        system:masters
```
The clusterrole adds the requisite permission.